Kitodo/src/main/java/org/kitodo/export/ExportDms.java

```diff
@@ -15,6 +15,7 @@
 import java.io.IOException;
 import java.net.URI;
 import java.net.URISyntaxException;
+import java.rmi.server.ExportException;
```

Comment:
Why do you not create a new exception class as this [class ](https://docs.oracle.com/javase/7/docs/api/java/rmi/server/ExportException.html)has nothing to do with Kitodo.Production export.